Detectors/TPC/qc/include/TPCQC/PID.h

```diff
@@ -70,7 +70,8 @@ class PID
 
   // To set the elementary track cuts
   void setPIDCuts(int minnCls = 60, float absTgl = 1., float mindEdxTot = 10.0,
-                  float maxdEdxTot = 70., float minpTPC = 0.05, float maxpTPC = 20., float minpTPCMIPs = 0.45, float maxpTPCMIPs = 0.55)
+                  float maxdEdxTot = 70., float minpTPC = 0.05, float maxpTPC = 20., float minpTPCMIPs = 0.45,
+                  float maxpTPCMIPs = 0.55, int CreateCanvasI = 1)
```

Comment:
Naming convention, more below.
```suggestion
                  float maxpTPCMIPs = 0.55, int createCanvasI = 1)
```